### PR TITLE
Small docs cleanups

### PR DESCRIPTION
**Pull request by Kashomon**
**What this PR does / why we need it**:

Readme: Fix terrraform -> terraform
Machines: replace tabs with spaces

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

Cleanup

@kubernetes/kube-deploy-reviewers
@jessicaochen

### CODE REVIEW

File: clusterctl/README.md [hunk 1]
```diff
@@ -25,7 +24,7 @@ TBD
 1. Create a `cluster.yaml`, `machines.yaml` and `provider-components.yaml` files configured for your cluster. See the provider specific templates and generation tools at `$GOPATH/src/sigs.k8s.io/cluster-api/clusterctl/examples/<provider>`. 
 2. Create a cluster 
 ```
-clusterctl create cluster -provider [google/terrraform] -c cluster.yaml -m machines.yaml -p provider-components.yaml
+clusterctl create cluster -provider [google/terraform] -c cluster.yaml -m machines.yaml -p provider-components.yaml
```
Comment (k4leung4) [inline clusterctl/README.md]: it should actually be vsphere, not terraform
https://github.com/kubernetes-sigs/cluster-api/blob/master/clusterctl/cmd/create_cluster.go#L146

Comment (k4leung4) [reply clusterctl/README.md]: and it should --provider

Comment (Kashomon) [reply clusterctl/README.md]: Done

**Pull request merged**